### ISSUE TITLE
Home page home icon add alt attribute

### ISSUE THREAD

**Issue by adamjarling** [samvera/hyrax#2979]
### Descriptive summary

From the Accessibility360 report:
> The "home" image link is not labeled correctly.
> Adjust the alt attribute to read "DCE home". 

Assume this is the link in screenshot below:

![accessibility-home-icon](https://user-images.githubusercontent.com/3020266/39435192-5a9ef1e2-4c60-11e8-939f-4983a37d1534.png)

### Rationale

Improved accessibility.

### Expected behavior

`alt` attribute is added to home icon image.

### Actual behavior

`alt` attribute currently not on home icon image.
